demo/social/social_queries.py

```diff
@@ -180,6 +180,25 @@
     expected_result=[]
 )
 
+post_delete_label_query = QueryInfo(
+    query="""MATCH (p:person) RETURN p.name""",
+    description='Retrieve all nodes with person label',
+    max_run_time_ms=0.5,
+    expected_result=[['Boaz Arad'],
+                     ['Valerie Abigail Arad'],
+                     ['Ori Laslo'],
+                     ['Shelly Laslo Rooz'],
+                     ['Ailon Velger'],
+                     ['Noam Nativ'],
+                     ['Jane Chernomorin'],
+                     ['Alon Fital'],
+                     ['Mor Yesharim'],
+                     ['Gal Derriere'],
+                     ['Lucy Yanfital'],
+                     ['Tal Doron'],
+                     ['Omri Traub']]
+)
+
```

Comment:
Prior to the change in this PR, this test would fail after the deletion in test 13. The deleted node would be replaced by 'Japan', which would then be found in this label scan.